### ISSUE TITLE
Pop out manager not working for MSFS ICON A5

### ISSUE THREAD

**Issue by rockywaves** [hawkeye-stan/msfs-popout-panel-manager#53]
I am running Simstrumentation GX3 panel in AM 4.  In the Pop out manager, I have ticked the check box for "touch enable".  Touch is working on the GX3 panels for buttons and knobs, but there are no changes in the sim panel, or the pop panel on my touch screen.  This suggests that the Simstrumentation GX3 does not work for the ICON A5, but my that is my non-technical guess.  The buttons on the pop out on the touch screen do not work either.   Has anyone successfully configured the Pop out Manager/Air Manager 4 to work with the ICON A5?

I do have the Pop out Manager and Air Manager working successfully with C172 Garmin 1000 NXi, so I think I know how to configure the Pop out Manager and Air Manager properly.

**Comment by hawkeye-stan** [comment]
Just tested the instrumentation touch capability and it seems to work. You've to allow click through for GX3 Panel in order to allow touch screen to work in Air Manager. Unfortunately, the buttons on the panel bezel doesn't seemed to work for me. You can try contacting Crunchmeister71 (the GX3 panel creator) at flightsimulator.com forum and he might be able to help you out.

<img width="304" alt="image" src="https://user-images.githubusercontent.com/72271266/211428100-5905a0d9-20f3-456d-bb8a-73d7a57698b0.png">

Stanley

**Comment by rockywaves** [comment]
Good morning Stanley,

Thank you for your support.  I have done further testing and found that the
instrument pop capability will work if I check both boxes for full screen
mode and touch enabled.  If I reduce the instrument pop out to any size
less than full screen, touch will no longer work.  Is there an explanation
for this?  I have all the preference settings in the pop manager turned off
for testing purposes, but should that make a difference?

Bob

**Comment by hawkeye-stan** [comment]
Full screen and custom panel window size should not make a difference. I've tested the instrumentation panel and G3X bezel for both full screen and in window. The only thing I can think of is somehow Air Manager G3X bezel is blocking mouse click and it is not passing the click through to the pop out instrumentation behind.

One thing to test is reverse the order. In Air Manager, uncheck "Always on Top" for G3X bezel. And in POPM check "Always on Top" for the instrumentation panel. This will make the instrumentation panel stay on top of Air Manager. If touch is working, then we know Air Manager G3X bezel could be causing the issue.

Stanley

**Comment by rockywaves** [comment]
I did not have AM running when running at the time.  I did have all the POM
preference setting unchecked at the time. Are there specific setting that
should on as default?  I did get an answer from Crunch, and learn that MSFS
2020 A5 Garmin is Aera 796.  I will try both your recommendations and let
you know what I discover.

**Comment by rockywaves** [comment]
Stan,  I abandoned further testing of ICON A5 instrument plan because
Crunch said that none of Sinstrumentation Garmin panels will work on the
A5..  I continued testing GX3 bezel and instrument panel in a different
plan (CubCrafter's NXCub) and discovered an issue which I feel might be a
Pop out manager problem.   I have touch enabled and always on top checked
and everything works great until I check the hide title bar.  Then the top
row of buttons no longer function with touch, but part of the panel below
the top row still functions properly.
[image: image.png]

Bob
